resources/views/member/partials/notices.blade.php

```diff
@@ -1,6 +1,7 @@
 @if ($member->memberRequest()->exists() && $member->memberRequest->approved_at == null)
     <div class="alert alert-warning">
-       This member's status request is currently pending. Some details, like teamspeak activity, may not be available until the request is approved.
+        This member's status request is currently pending. Some details, like teamspeak activity, may not be available
```

Comment:
Drop the TeamSpeak reference?

Reply:
Let's do those cleanups in a separate PR